### PR TITLE
Autogenerated/apple.boot camp drivers/b5211d2d b630 4028 b631 1719cad5afa9

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113306&drop=dogfoodAlpha

### CODE REVIEW

**Comment by upintheairsheep** [comment]
Would like a validation screenshot.

File: manifests/a/Apple/BootCampDrivers/5.1.5769/Apple.BootCampDrivers.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageIdentifier: Apple.BootCampDrivers
+PackageVersion: 5.1.5769
+Installers:
+- InstallerUrl: https://download.info.apple.com/Mac_OS_X/031-30890-20150812-ea191174-4130-11e5-a125-930911ba098f/bootcamp5.1.5769.zip
+  Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: BootCamp\setup.exe
+  InstallerSha256: FD657F2DABAD7BB08BFC6F98FAD19487CAABA15FB1BF3C1C9C59C36FF25690A2
```
Comment (ItzLevvie) [inline manifests/a/Apple/BootCampDrivers/5.1.5769/Apple.BootCampDrivers.installer.yaml:13]:
```suggestion
  InstallerSha256: 4ede2c8ef240708c850237a3e5911094ed6adae1734258e4639bc9069a814b1e
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
[Policy] Hardware

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by ItzLevvie** [comment]
[Policy] Hardware

**Comment by denelon** [comment]
Supporting custom hardware in validation is "Not Planned".
* https://github.com/microsoft/winget-pkgs/issues/199504

**Pull request closed without merging**